DEV_NOTES.md

```
@@ -5,4 +5,8 @@ bunx expo install expo@next --fix
 
 ## Animations
 
-https://docs.swmansion.com/react-native-reanimated/examples/switch
\ No newline at end of file
+https://docs.swmansion.com/react-native-reanimated/examples/switch
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)